src/grandorgue/sound/GOSoundProviderWave.cpp

```diff
@@ -201,68 +202,86 @@ void GOSoundProviderWave::LoadFromOneFile(
   bool use_pitch,
   unsigned loop_crossfade_length,
   unsigned max_released_time) {
-  GOWave wave;
+  // if an exception occures during Open(), it alteady contains the file name
```

Comment:
// if an exception occurs during Open(), it already contains the file name

Reply:
Fixed